go.mod

```diff
@@ -8,49 +8,51 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.0
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/google/go-github/v29 v29.0.3
+	github.com/google/go-github/v67 v67.0.0
 	github.com/gorilla/handlers v1.4.2
-	github.com/gorilla/mux v1.8.0
+	github.com/gorilla/mux v1.8.1
 	github.com/pkg/errors v0.9.1
-	github.com/rancher/apiserver v0.0.0-20240124133512-e93bfa07e5f1
-	github.com/rancher/wrangler/v3 v3.0.0-rc2
+	github.com/rancher/apiserver v0.5.1
+	github.com/rancher/wrangler/v3 v3.0.1-rc.2
```

Comment:
Can you bump the version here to Wrangler's latest version v3.1.0, or v3.0.1 if that does not work? 

ref: https://github.com/rancher/wrangler/tags

Reply:
I set it to the same version as rancher/apiserver is using, but I guess v3.1 should be OK. 

I don't know why someone tagged it this way, but somehow the latest 3.0.1 RC tags are actually NEWER than 3.0.1:
![image](https://github.com/user-attachments/assets/ec73c2ee-bd36-4a7e-84e0-e3cd97b60b86)

Reply:
cc @tomleb can you comment on why we have rc tags that are newer than the patch release that they are an rc for?

Reply:
If we needed new RCs for 3.0.1, those should have been 3.0.2-rc.1 and 3.0.2-rc.2. From a semver perspective they are "lower" than 3.0.1 despite being newer.